pkg/builder/quarkus.go

```diff
@@ -206,6 +211,7 @@ func GenerateQuarkusProjectCommon(runtimeVersion string, quarkusVersion string,
 				},
 			},
 		},
+		jib.JibMavenPlugin(layers),
```

Comment:
Not sure to understand this. It's returning a plugin object, but it seems nobody is using it.

Reply:
The plugin object is added to be used in the jib task code `pkg/builder/jib.go` at publish time.